controllers/admin/AdminShopGroupController.php

```diff
@@ -187,6 +187,15 @@ public function renderForm()
                     'name' => 'name',
                     'required' => true,
                 ],
+                [
+                    'type' => 'color',
+                    'label' => $this->trans('Color', [], 'Admin.Catalog.Feature'),
+                    'name' => 'color',
+                    'desc' => [
+                        $this->trans('It will only apply to this group of shops, each store keeps its individual color.', [], 'Admin.Shopparameters.Feature'),
```

Comment:
```suggestion
                        $this->trans('It will only be applied to this group of shops, each store will keep its individual color.', [], 'Admin.Shopparameters.Feature'),
```